src/BaGet.Web/Controllers/IndexController.cs

```diff
@@ -10,31 +12,33 @@ namespace BaGet.Web.Controllers
     /// </summary>
     public class IndexController : Controller
     {
+        private IEnumerable<ServiceResource> ServiceWithAliases(string name, string url, params string[] versions) {
+            foreach(var version in versions) {
+                string fullname = string.IsNullOrEmpty(version) ? name : name + "/" + version;
+                yield return new ServiceResource(fullname, url);
+            }
+        }
+
         // GET v3/index
         [HttpGet]
         public object Get()
         {
             // Documentation: https://docs.microsoft.com/en-us/nuget/api/overview
             // NuGet.org: https://api.nuget.org/v3-index/index.json
-
-            // TODO: Client is picky about versions' formatting. Needs more research. See:
-            // https://github.com/NuGet/NuGet.Client/blob/67d34ee8158597377fb26e06987ad6ad40a1b09b/src/NuGet.Core/NuGet.Protocol/Constants.cs
+            
+            string version = "3.0.0";
             return new
             {
-                Version = "3.0.0-beta.1",
-                Resources = new[]
-                {
-                    // Required
-                    new ServiceResource("PackagePublish/Versioned", Url.PackagePublish()),
-                    new ServiceResource("SearchQueryService/Versioned", Url.PackageSearch()),
-                    new ServiceResource("RegistrationsBaseUrl/Versioned", Url.RegistrationsBase()),
-                    new ServiceResource("PackageBaseAddress/Versioned", Url.PackageBase()),
-
-                    // Optional
-                    new ServiceResource("SearchAutocompleteService/Versioned", Url.PackageAutocomplete()),
-                    //new ServiceResource("ReportAbuseUriTemplate/3.0.0-rc", new Uri("https://google.com")),
-                    //new ServiceResource("Catalog/3.0.0", new Uri("https://google.com"))
-                }
+                Version = version,

```

Comment:
The `version` variable seems unnecessary now. How about just `Version` = "3.0.0",`?